docs/kyma/docs/025-details-local-reinstallation.md

```diff
@@ -2,6 +2,9 @@
 title: Reinstall Kyma
 type: Details
 ---
+
+To work with Kyma, use only the provided installation and deinstallation scripts. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command.
```

Comment:
This note will be placed in the main "local deployment" doc. See https://github.com/kyma-project/kyma/pull/137

Reply:
Sure so, as discussed, I will only remove the prerequisites doc and the way you put the details from it into the existing docs, I leave up to you.